src/store/conversationsStore.js

```diff
@@ -122,6 +137,55 @@ const mutations = {
 	addConversation(state, conversation) {
 		Vue.set(state.conversations, conversation.token, conversation)
 	},
+
+	/**
+	 * Add new conversations
+	 *
+	 * @param {object} state the state
+	 * @param {object[]} conversations new conversations list
+	 */
+	addConversations(state, conversations) {
+		/**
+		 * Apply mutations to object based on new object:
+		 *
+		 * @param {object} target target object
+		 * @param {object} newObject new object to get changes from
+		 */
+		const applySoftObjectUpdates = (target, newObject) => {
+			const isObject = (value) => typeof value === 'object' && !Array.isArray(value)
+
+			// Delete removed properties
+			for (const key of Object.keys(target)) {
+				if (!Object.hasOwn(newObject, key)) {
+					del(target, key)
+				}
+			}
+
+			// Add new properties and update old ones
+			for (const key of Object.keys(newObject)) {
+				if (!Object.hasOwn(target, key)) {
+					// Add new property
+					set(target, key, newObject[key])
```

Comment:
```suggestion
					Vue.set(target, key, newObject[key])
```
maybe a matter of preference, but in order to keep consistent I'd write with like all the other mutations (or we could rewrite all the others later)

Reply:
Vue.set is deprecated

Reply:
Then let's raise a follow up later

Reply:
Hm, Compat built has only the old `Vue.set` and not the new `set` from `2.7`.
Then it is the opposite for easy migration with the Compat
https://v3-migration.vuejs.org/migration-build.html#feature-reference